docs/competition/teams/role-descriptions/reception.md

```diff
@@ -51,3 +51,4 @@ You should have the following items present for your duties. If you do not, plea
 * Spare MCFs to allow visitors to enter.
 * Pens to fill in MCFs.
 * Wristbands to give to entrants.
+* Competition leaflets to give to teams
```

Comment:
What does this mean? Is this the competitor booklets?

Reply:
I've changed the name to be literally 'Competitor Booklets' in 27a6f51